frontend/src/citizen-frontend/navigation/MobileNav.tsx

```diff
@@ -142,7 +142,7 @@ const BottomBar = styled.nav`
   height: ${mobileBottomNavHeight}px;
   width: 100%;
   padding: ${defaultMargins.xs};
-  position: sticky;
+  position: fixed;
```

Comment:
Using `position: fixed` means that 60px of content on every page is hidden behind the bottom bar. The problem in messages page is that the `FullHeightContainer` container in `MessagesPage.tsx` has a fixed height so the bottom bar is positioned after container but the actual contents inside the container overflow and are shown under the bottom bar. It seems that removing `height: calc(100vh - ${headerHeightDesktop}px);` fixes it and doesn't have an effect on anything else. It must have been put there for some reason at some point but whatever the reason was it doesn't seem to work anymore.